intercom_flutter_platform_interface/lib/intercom_flutter_platform_interface.dart

```diff
@@ -278,4 +278,9 @@ abstract class IntercomFlutterPlatform extends PlatformInterface {
   Future<void> displayConversation(String conversationId) {
     throw UnimplementedError('displayConversation() has not been implemented.');
   }
+
+  /// To display an Activity with your Messages content.
```

Comment:
"Submit, view, and get updates on all of your tickets from one centralized place in the Messenger."